internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -57,3 +59,21 @@ func (c *CodePipeline) GetPipeline(pipelineName string) (*Pipeline, error) {
 
 	return pipeline, nil
 }
+
+// ListPipelines retrieves summaries of all pipelines for a project
+func (c *CodePipeline) ListPipelines() ([]string, error) {
```

Comment:
Won't this get *all* pipelines in an account/region, not just a particular project?

If we want to limit it to just the pipelines in this project - we can query ResourceGroups for resources with the `ecs-cli-project` tag (value == whatever this project is) and filter by the resource type: `AWS::CodePipeline::Pipeline`

I think we do this right now with alarms!

Reply:
Ah okay. will fix.